lighting/raymarch.glsl

```diff
@@ -42,8 +42,8 @@ license:
 #define RAYMARCH_CAMERA_FOV 60.0
 #endif
 
-#ifndef ENG_RAYMARCH
-#define ENG_RAYMARCH
```

Comment:
I wasn't sure if this was a typo of there is a bigger plan behind this one

Reply:
I'm not sure I've done this 😅
There HUGE plans, but not on this specifically 😂
Feel free to rename.